armi/reactor/converters/tests/test_uniformMesh.py

```diff
@@ -84,7 +82,8 @@ def test_makeAssemWithUniformMesh(self):
         )
 
         prevB = None
-        for newB, sourceB in zip(newAssem, sourceAssem):
+        for newB in newAssem:
+            sourceB = sourceAssem.getBlockAtElevation(newB.p.z)
```

Comment:
This change was needed because with the new blueprints def, the `newAssem` didn't have the same number of blocks as the `sourceAssem`.